src/app/learnocaml_student_view.ml

```diff
@@ -446,20 +477,19 @@ let update_report_tab exo ans =
       Manip.replaceChildren El.Tabs.(report.tab)
         [H.div [H.txt [%i"No report available"]]]
 
-let update_tabs meta exo ans =
+let update_tabs meta exo ans syn =
   update_text_tab meta exo;
-  match ans with
-  | None -> ()
-  | Some ans ->
+  update_draft_tab syn;
+  Option.iter (fun ans ->
       update_report_tab exo ans;
-      update_answer_tab ans.Answer.solution
+      update_answer_tab ans.Answer.solution) ans
 
 let () =
   run_async_with_log @@ fun () ->
   (* set_string_translations (); *)
   (* Manip.setInnerText El.version ("v."^Learnocaml_api.version); *)
   Learnocaml_local_storage.init ();
-  (match Js_utils.get_lang() with Some l -> Ocplib_i18n.set_lang l | None -> ());
+  Js_utils.get_lang () |> Option.iter (fun l -> Ocplib_i18n.set_lang l);
```

Comment:
Alright, this saves a few character, but to further improve readability I propose to remove line noise and the unneeded anonymous function:
```suggestion
  Option.iter Ocplib_i18n.set_lang (Js_utils.get_lang ());
```

Reply:
Good point: the eta-expansion was unneeded here!